app/views/handovers/legacy_index.html.erb

```diff
@@ -2,6 +2,12 @@
   <%= render '/layouts/prison_switcher' %>
 <% end %>
 
+<% if session[:new_handovers_ui].present? %>
```

Comment:
Non-blocking comment, rather than testing presence, is it not better to ensure that the value of `session[:new_handovers_ui]` is `true`? That way we can implement a true/false switch for the feature flags in future (should we wish to) rather than relying on deletion of the param

Reply:
Done.